src/Libraries/RevitNodes/Elements/Views/View.cs

```diff
@@ -200,16 +199,18 @@ public IEnumerable<Revit.Filter.ParameterFilterElement> Filters
         ///     This behavior will persist even if View has a View Template applied which normally would prevent user from adding Filters without first
         ///     disabling or modifying the View Template.
         /// </summary>
-        /// <param name="parameterFilter">Parameter filter</param>
-        /// <param name="overrides">overrides settings</param>
+        /// <param name="parameterFilter">Parameter Filter</param>
+        /// <param name="overrides">Graphic Overrides Settings</param>
+        /// <param name="hide">If True given Filter will be hidden.</param>
         /// <returns name="view">View</returns>
-        public Element SetFilterOverrides(Revit.Filter.ParameterFilterElement parameterFilter, Revit.Filter.OverrideGraphicSettings overrides)
+        public Revit.Elements.Views.View SetFilterOverrides(Revit.Filter.ParameterFilterElement parameterFilter, Revit.Filter.OverrideGraphicSettings overrides, bool hide = false)
```

Comment:
please see above. You are basically skipping the `ToDSType()` method which takes the `View` object and casts it back into the `View`...its redundant.